domains/daudix.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "daudix",
+        "email": "ddaudix@gmail.com"
+    },
+    "record": {
+        "CNAME": "pages.daudix.codeberg.page"
```

Comment:
I cannot reach your site
"Your connection is not private" (and the site uses HSTS so i can't access it using http)